ddp/ddp_pipeline/src/test/java/org/mskcc/cmo/ks/ddp/pipeline/util/DDPUtilsTest.java

```diff
@@ -268,68 +297,6 @@ public void resolveOsMonthsTest() throws ParseException {
         resolveOsMonthsAndAssert("LIVING", "2013-07-19", "ignore", "2016-06-08", "Fri, 20 Jan 2017 16:52:02 GMT", "NA");
     }
 
-    /**
-     * Tests for resolvePatientAgeAtDiagnosis().
-     * NOTE: Darwin pipeline outputs patient age at diagnosis ("AGE") to darwin/darwin_age.txt.
-     *      As of 2018/06/05, this file is only used for genie and is not generated
-     *      by the DDP pipeline yet. At this time there is no need to implement the
-     *      DDP equivalent of the darwin/darwin_age.txt file as it is still being
-     *      generated by the Darwin pipeline.
-     *
-     *      The Darwin pipeline calculates "AGE" as the years since birth:
-     *              (currentYear - patientBirthYear)
-     *      This behavior is mirrored in DDPUtils.resolvePatientAgeAtDiagnosis() and
-     *      will not be modified until further notice.
-     *
-     *      Eventually the age of diagnosis will be calculated on a per-sample basis, using
```

Comment:
I remember this discussion. There were subtleties about defining when a patient was first diagnosed because a patient may have several different/independent cancer diagnosis. We had talked about grouping samples based on the type of cancer and possibly having multiple first diagnosis dates. Maybe we are no longer going to try to do this in the future - and to only report the very first day a patient had *any* sample sequenced. But I think maybe we should ask whether a note about this intention should be made anywhere (perhaps on a card or a roadmap planning document) Maybe @n1zea144 can comment here.

Reply:
`resolvePatientAgeAtDiagnosis()` function was removed in this PR because it's not called anywhere (only in this test file). Marking resolved